features.yml

```diff
@@ -921,6 +927,22 @@
   tags:
     - css
     - css_selectors
+ 
```

Comment:
Just putting this here as a suggestion. Not sure if you want to include it or not. Also, unclear where the caniuse values come from.